Formula/minizip-ng.rb

```diff
@@ -19,9 +19,16 @@ class MinizipNg < Formula
   depends_on "zstd"
 
   uses_from_macos "bzip2"
-  uses_from_macos "libiconv"
   uses_from_macos "zlib"
 
+  on_linux do
+    depends_on "openssl@1.1"
```

Comment:
We should probably try `openssl@3.0`, since this shouldn't have mixed versions in its dep tree.